src/scripts/pokeballs/Pokeballs.ts

```diff
@@ -25,6 +25,15 @@ class Pokeballs implements Feature {
             new Pokeball(GameConstants.Pokeball.Ultraball, () => 10, 750, '+10% chance to catch'),
             new Pokeball(GameConstants.Pokeball.Masterball, () => 100, 500, '100% chance to catch'),
             new Pokeball(GameConstants.Pokeball.Fastball, () => 0, 500, 'Reduced catch time', new RouteKillRequirement(10, GameConstants.Region.johto, 34)),
+            new Pokeball(GameConstants.Pokeball.Moonball, () => {
+                const moonCycleMod = MoonCycle.currentMoonCyclePhase();
+                const moonCycleBonus = (4 - Math.abs((moonCycleMod % 8) - 4)) * 5;
```

Comment:
This caps out at 15%, yes?

Reply:
No, it currently caps out at 20%, this was lengthy discussed in the Dev-Help Chat, where the conclusion was that 20% once every 8 days was fine. But I'm more then happy to adjust it down to capping out at 12%, or going from 3% to 15% instead of 0% to 20%.